.github/workflows/dotnet-ci.yml

```
@@ -71,3 +71,33 @@
+        with:
+          dotnet-version: "8.0.x"
+
+      - name: Restore dependencies
+        run: dotnet restore TaskForge.NET/TaskForge.NET.sln
+
+      - name: Publish .NET MVC App
+        run: |
+          dotnet publish TaskForge.NET/TaskForge.NET.csproj -c Release -o ./publish
+
+      - name: Run Railway CLI
+        uses: ghcr.io/railwayapp/cli:latest
```

actionlint (1.7.4)

specifying action "ghcr.io/railwayapp/cli:latest" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)